src/data_race.rs

```diff
@@ -24,7 +24,7 @@
 //! because it only re-uses vector indexes once all currently-active (not-terminated) threads have an internal
 //! vector clock that happens-after the join operation of the candidate thread. Threads that have not been joined
 //! on are not considered. Since the thread's vector clock will only increase and a data-race implies that
-//! there is some index x where clock[x] > thread_clock, when this is true clock[candidate-idx] > thread_clock
+//! there is some index x where clock\[x\] > thread_clock, when this is true clock[candidate-idx] > thread_clock
```

Comment:
```suggestion
//! there is some index x where clock\[x\] > thread_clock, when this is true clock\[candidate-idx\] > thread_clock
```
That just seems more consistent, even if rustdoc does not require it...

Reply:
oop, yeah
it probably failed to detect it

Reply:
I also considered allowing this particular warning, because it seemed extremely pedantic

Reply:
Well, this is markdown, and [...] has special meaning there, so -- makes sense to me.